bnd-maven-plugin-parent/bnd-maven-plugin-dependencies/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin-parent</artifactId>
+        <version>2.5.0-SNAPSHOT</version>
```

Comment:
Well the master branch is about building the upcoming release which is 3.0. This would mean that a bnd-maven-plugin built in this branch would build against the bndlib also built in this branch which is bndlib 3.0. We need to have this in the master branch to build the 3.0 version of the bnd-maven-plugin for 3.0. But you also have an additional requirement which is to build a version for bndlib 2.4.1 and that should be built out of another branch. That branch does not have to be in this repo  but it certainly can be. If it is in this repo, then the bnd-maven-plugin based upon bndlib 2.4.1 should be built in next or a new branch off next (which is probably safest).